src/subscriber.ts

```diff
@@ -170,52 +183,113 @@ export class Subscriber {
 
     private async _sessionOfflineHandler(header: Header): Promise<void> {
```

Comment:
Maybe we could rename this method given that we are not only handling session offline here, wdyt?

Reply:
I totally agree